scripts/ricardeos/ricardeos.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+
+import json
+import sys
+import os.path
+import fnmatch
+
+def add_ricardian_contracts_to_actions(source_abi_directory, contract_name, abi_actions):
+    abi_actions_with_ricardian_contracts = []
+
+    for abi_action in abi_actions:
+        action_name = abi_action["name"]
+        contract_action_filename = f'{contract_name}-{action_name}-rc.md'
```

Comment:
This will need to be the old `format` since they use 3.3 in some of their builders.